pfcpiface/upf.go

```diff
@@ -116,17 +116,28 @@ func NewUPF(conf *Conf, fp datapath) *upf {
 		nodeID = hosts[0]
 	}
 
+	var dnns []string
+	var ippoolCidr string
+
+	// Extract DNN names and UEIPPool from the new DnnList structure
+	for _, dnnInfo := range conf.CPIface.DnnList {
+		dnns = append(dnns, dnnInfo.DNN)
+		if ippoolCidr == "" {
+			ippoolCidr = dnnInfo.UEIPPool
```

Comment:
In relation to my previous comment, here it actually looks like the UE IP pool for the last DNN is used, correct? or am I overlooking something?